modules/distr-tracing-rn-fixed-issues.adoc

```diff
@@ -0,0 +1,29 @@
+////
+Module included in the following assemblies:
+* distr-tracing-release-notes.adoc
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+[id="distr-tracing-rn-fixed-issues_{context}"]
+= Distributed tracing fixed issues
+////
+Provide the following info for each issue if possible:
+Consequence - What user action or situation would make this problem appear  (If you have the foo option enabled and did x)? What did the customer experience as a result of the issue? What was the symptom?
+Cause - Why did this happen?
+Fix - What did we change to fix the problem?
+Result - How has the behavior changed as a result?  Try to avoid “It is fixed” or “The issue is resolved” or “The error no longer presents”.
+////
+
+* link:https://issues.redhat.com/browse/TRACING-2009[TRACING-2009] The Jaeger Operator has been updated to include support for the Strimzi Kafka Operator 0.23.0.
+
+* link:https://issues.redhat.com/browse/TRACING-1907[TRACING-1907] The Jaeger agent sidecar injection was failing due to missing config maps in the application namespace. The config maps were getting automatically deleted due to an incorrect `OwnerReference` field setting and as a result, the application pods were not moving past the "ContainerCreating" stage. The incorrect settings have been removed.
```

Comment:
missing comma. 

`...field setting, and as a result, the application pods were not moving past...`